pkg/xen-tools/patches-4.19.0-rc4-v3/17-enable-qemu-vhost-vsock.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/tools/qemu-xen/configure b/tools/qemu-xen/configure
+index a62a3e6..c1b24ef 100755
+--- a/tools/qemu-xen/configure
++++ b/tools/qemu-xen/configure
+@@ -896,6 +896,10 @@ for opt do
+   ;;
+   --disable-vfio-user-server) vfio_user_server="disabled"
+   ;;
++--enable-vhost-scsi) vhost_scsi="yes"
```

Comment:
@roja-zededa , these options were moved to Kconfig, right? How just defining these variables here are activating them? You need to add them to the corresponding device/target, for instance: add CONFIG_VHOST_SCSI=y to configs/devices/i386-softmmu/default.mak (for x86)... take a look on https://www.qemu.org/docs/master/devel/kconfig.html

Reply:
Updated the KConfig file under QEMU-xen/hw/virtio/Kconfig to enable vhost-vsock and vhost-scsi by default so removing the corresponding --enable options from pkg/xen-tools/Dockerfile

Reply:
Removed this patch as vhost-vsock and vhost-scsi are enabled by default. Also, we need to remove the corresponding --enable options from Xen-tools/Dockerfile because of the default enablement.